devtools/create_achievements/gen/steam-1259140.ini

```diff
@@ -121,7 +121,7 @@ item_39_title=Of blood and water.
 item_39_comment=I am the Lady On The Wall.
 item_40_id=ACH_ENDING_LUZIA
 item_40_title=My Big Sister
-item_40_comment=Our tale is just begining.
+item_40_comment=Our tale is just beginning.
```

Comment:
This was just extracted from the steam achievements I believe so we may want to keep it for consistency, but I don't have a strong opinion.